content/strategy-goals/strategy/cloud/delivery/index.md

```diff
@@ -1,183 +1,33 @@
+This page outlines the short-term strategy for the [Delivery team](../../../../departments/product-engineering/engineering/cloud/delivery/index.md). You can find the overview of Sourcegraph Product and Engineering FY23 Q2 focus [here](../../../../departments/product-engineering/strategy-goals/index.md#fy23-q2-focus)
 
 ---
+## Q2 Objective and Key Results
 
-# Mission & Vision
+### Problem Area: Improve customer admin experience with deploying, upgrading, and maintaining self-hosted Sourcegraph instances. 
 
-## Mission
+Our goal is to ensure any Sourcegraph prospect or customer to use Sourcegraph in a way fits within their environment and allows them to easily access the value that our product provides. Make it as easy as possible for customers to access the value of Sourcegraph's latest developments through a simple upgrade process
 
-Enable any Sourcegraph prospect or customer to use Sourcegraph in a way fits within their environment and allows them to easily access the value that our product provides. Make it as easy as possible for customers to access the value of Sourcegraph's latest developments through a simple upgrade process.
+- Key Result 1: Reduce the number of support tickets raised to Delivery team about deployment resourcing needs by shipping improved documentation for Resource Estimation based on customer product usage.
+- Key Result 2: Define a data metric inventory and cross-functional DRIs required for long term Resource Estimator tooling improvements.
+- Key Result 3: Complete an RFC for a proposal to establish a centralized location to store data about customer architecture and resourcing needs.
+- Key Result 4: Interview 5 or more eneterpise customers who have either deployed Sourcegraph to production or ran an upgrade in the last 3 quarters to understand their pain points
+- Key Result 4: Ship customer research insights document synthesizing customer research findings. 
 
-## Problem Space
+Within this area, in Q2 we are not working on:
+- Shipping improvements to Resource Estimator tool itself 
+- Support for non-sequential version upgrades 
 
-While there are seemingly 3 problems here, the reality is that it narrows down to one.
-(This is a slight generalization as there are things that can be done to improve each individually, but the root of the majority of the issues lie in one place, and we consider that to be our opportunity to deliver maximum value)
+### Problem Area: Build customer trust by improving internal troubleshooting processes
 
-#### 1. POCs
+We want to streamline our troubleshooting processes for our Customer Engineering and Customer Support teams to ensure we can reduce any friction customers may face during setup, upgrade, or maintenance of a self-hosted Sourcegraph instance. Our goal is to help immprove the customer experience and reduce the time to resolve issues. 
 
-- POCs take one of two forms:
```

Comment:
@kevinwojo and @caugustus-sourcegraph I removed a lot of this as it either doesn't seem relevant for this page and should be elsewhere (our team page) or because we haven't written a committed LT roadmap so should not be a calling that our publicly. LMK what you think